tests/mcp_structure_test/mcp/gcp_ae_locals.tf

```diff
@@ -0,0 +1,89 @@
+locals {
```

Comment:
Why is there code in the tests directory like this? Won't it override the real code in the module we're trying to test?